CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
 
 _No documentation available about unreleased changes as of yet._
 
+## [0.2.0] - 2019-04-13
```

Comment:
This shouldn't be part of this PR - other changes have happened since as well, so it makes more sense to add the changelog separately for all changes in one go.

Reply:
Will remove this definitely